tests/cases/fourslash/importFixesWithPackageJsonInSideAnotherPackage.ts

```diff
@@ -0,0 +1,36 @@
+/// <reference path="fourslash.ts" />
+
+// @Filename: /project/tsconfig.json
+////{
+////  "compilerOptions": {
+////    "jsx": "react",
+////    "jsxFactory": "h"
+////  }
+////}
+
+// @Filename: /project/app.tsx
+////const state = useMemo(() => 'Hello', []);
+
+// @Filename: /project/component.tsx
+////import { useEffect } from "preact/hooks";
```

Comment:
this line seems kinda useless without all the context from the repro, should it be removed or does it still serve a purpose?

Reply:
Its used to include preact/hooks/src/index.d.ts to be included in the program

Reply:
ah, right